src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -94,7 +122,12 @@ const ChatItemMyTextV2 = ({
       onSwipeableOpen={onSwipeToReply}
       renderLeftActions={replyIcon}>
       <View style={containerStyle(true, isReplyPrompt)}>
-        <Animated.View style={[styles.wrapper, animatedBubbleStyle]}>
+        <Animated.View
+          style={[
+            styles.wrapper,
+            animatedBubbleStyle as ViewStyle,
+            animatedPulseStyle as ViewStyle
+          ]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `animatedBubbleStyle` and `animatedPulseStyle` are now casted to `ViewStyle`. This could potentially hide type errors. Please ensure that these styles are indeed compatible with `ViewStyle`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->